src/fluree/json_ld/processor/api.cljc

```diff
@@ -0,0 +1,251 @@
+(ns fluree.json-ld.processor.api
+  (:refer-clojure :exclude [flatten])
+  (:require #?(:clj [jsonista.core :as json])
+            #?(:cljs ["jsonld" :as jldjs]))
+  #?(:clj (:import (com.apicatalog.jsonld JsonLd)
+                   (com.apicatalog.jsonld.document JsonDocument RdfDocument)
+                   (java.io StringReader))))
+
+#?(:clojure (set! *warn-on-reflection* true))
```

Comment:
What does `:clojure` do here? Is it equivalent to `:clj`? If so, might be better to just use `:clj` so it matches the same searches, etc.

Reply:
Good catch!